src/components/Link/Link.component.tsx

```
@@ -0,0 +1,23 @@
+        linkSize = 'sm',
+        backgroundColor = 'primary',
+        borderRadius = 'sm',
+        ...rest
+    } = props;
+    return (
+        <StyledLink
+            linkSize={linkSize}
+            backgroundColor={backgroundColor}
+            borderRadius={borderRadius}
+            {...rest}
+        ></StyledLink>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)